### ISSUE TITLE
Motor status

### ISSUE THREAD

**Issue by meguiraun** [mxcube/mxcube3#186]
For the motors/moveables we send data through the socketio. @marcus-oscarsson uses 'IDLE', 'BUSY', 'ABORT' in the beamline related signals. Looking at the ```AbstracMotor``` the following is defined:
```
    (NOTINITIALIZED, UNUSABLE, READY, MOVESTARTED, MOVING, ONLIMIT) = (0,1,2,3,4,5)
    EXPORTER_TO_MOTOR_STATE = { "Invalid": NOTINITIALIZED,
                                "Fault": UNUSABLE,
                                "Ready": READY,
                                "Moving": MOVING,
                                "Created": NOTINITIALIZED,
                                "Initializing": NOTINITIALIZED,
                                "Unknown": UNUSABLE,
                                "LowLim": ONLIMIT,
                                "HighLim": ONLIMIT }
```
Should we agree on the keywords we send for all the moveables? I would vote for the ones appearing in ```AbstractMotor```, but I am not sure the reason for the difference, if any.

**Comment by marcus-oscarsson** [comment]
Hi Mikael,

We should definitely use the terminology developed in AbstractMotor, there is no good reason for the divergence from my side. The intention was never really to keep those keywords, I chose something I thought made sense for the moment, in order to get something working. We should perhaps also by convention use the constants defined in AbstractMotor instead of strings so that we avoid divergence.

**Comment by mguijarr** [comment]
+1 for using the constants from AbstractMotor

**Comment by meguiraun** [comment]
No sorries! You mean to send the status as integer (0, ... 5)? In fact, I would rather go with sending strings since they have a clear meaning. Though looks I am losing the vote! No problem at all.

**Comment by marcus-oscarsson** [comment]
Hey again,

I think there is maybe still some "definition" work to be done on the AbstractMotor I don't know in what extent its actually used today. So we need to make sure that the values defined corresponds to reality :). 

I don't really mind if we also define string constants as long as they correspond to the integer values. The benefit with string values are that it would be easier to debug the http requests. However I think we need a corresponding definition on the client side in anyways. So for me it does not really matter that much, is as you like :).

**Comment by marcus-oscarsson** [comment]
Resolved by #238 Motor State
